src/main/java/racingcar/RacingGameConsole.java

```diff
@@ -4,6 +4,7 @@
 
 public class RacingGameConsole {
 
+    private static final DrivingStrategyImpl DRIVING_STRATEGY = new DrivingStrategyImpl();
```

Comment:
변수의 타입은 구현체보다 추상 타입으로 선언하면 어떨까요?
```suggestion
    private static final DrivingStrategy DRIVING_STRATEGY = new DrivingStrategyImpl();
```

(참고만 해주세요)
개인적으로 ~Impl 접미사는 활용하지 않는 편인데요
단순 구현체라는 의미로는 클래스가 무엇을 뜻하는지 파악하기 어렵다는 점에서 입니다
List 라는 인터페이스의 구현체의 이름이 ListImpl이 아닌 ArrayList, LinkedList 등 클래스가 무엇인지 나타내고 있습니다
랜덤 숫자로 이동 가능 여부를 판단하니 클래스명에서 표현되면 좋을 것 같아요